ui/views.py

```diff
@@ -91,8 +92,14 @@ def get_context_data(self, **kwargs):
         video = context["object"]
         refresh_status(video)
         context['form'] = VideoForm(instance=video)
-        context['videofile'] = context['object'].videofile_set.filter(encoding=EncodingNames.HLS)
-        context["js_settings_json"] = json.dumps(default_js_settings(self.request))
+        try:
+            videofile = video.videofile_set.get(encoding=EncodingNames.HLS)
+        except ObjectDoesNotExist:
+            videofile = None
```

Comment:
When the video transcoding is not yet complete there will be no videofile object with encoding=HLS